rpcclient/evo_test.go

```diff
@@ -279,6 +333,7 @@ func TestProTxInfo(t *testing.T) {
 
 	proTxHash := "ec21749595a34d868cc366c0feefbd1cfaeb659c6acbc1e2e96fd1e714affa56"
 
+	client.httpClient.Transport = mockRoundTripperFunc(&btcjson.ProTxInfoResult{ProTxHash: proTxHash})
```

Comment:
`modifyThenCompareWithCliCommand` doesn't verify anything, we need to add some validation of results.